rfcs/text/0000-Incident-Response-Plan.md

```diff
@@ -0,0 +1,199 @@
+
+### Incident Severity Scale
+In additional to tracking incident rate of occurrence, mean time to detect, and mean time to repair, DCP employs a standard rating of incident severity that describes the impact of the incident to the DCP.
+
+The severity scale is determined by three general criteria:
+* __Confidentiality__. Is the data clinical or non-clinical, open access or managed access?
+* __Integrity__. Is the system producing correct results?
+* __Availability__. Is the system up and running?
+
+#### Incident Severity Ratings 
+* __High__ - DCP is no longer able to provide some critical services to users; sensitive identifiable data was accessed without authorization or exfiltrated; or recovery from the incident is unpredictable or not possible.
+* __Moderate__ - DCP has lost the ability to provide a critical service to a subset of users; non-sensitive data was accessed without authorization or exfiltrated; or recovery from the incident is predictable with additional resources.
+* __Low__ - Minimal effect; DCP can still provide all critical services to users but has lost efficiency; recovery from the incident is predictable with existing resources.
+
+### Incident Response Plan and Procedures
+#### Incident Response Lifecycle Overview
+The DCP IRT follows the incident response lifecycle described below.
+
+![Incident Response Lifecycle](https://drive.google.com/uc?export=view&id=1cTXM31emPeS2RiaSI6p8yLfyZG8a6_R3)
+
+##### Phase 1: Preparation
+Preventing incidents from occurring is a part of preparation. DCP personnel implement appropriate security controls as part of the overall DevSecOps program. Preparation activities related to incident response are as follows:
+* Establish an incident response capability
+* Develop incident response policies and procedures
+
+##### Phase 2: Detection and Analysis
+Incident identification occurs in this phase. The DCP IRT learns of an actual or suspected security incident. Incidents may be detected through many different means with varying levels of detail and veracity. Security incidents may be detected through some of the following examples:
+* End user identification
+* System administrator identification
+* Log review
+* Automated alerting tools
+* Locked out accounts
+* Pop-ups and redirected websites when browsing 
+* Unexpected software installs
+* Unexplained changes to files
+* Anomalies in normal network traffic patterns
+* Abnormal outbound traffic
+* Irregular access locations
+* Large number of requests for the same object or files
+* Suspicious activity on the network after-hours
+* Multiple failed login attempts
+* Unknown/unauthorized IP addresses
+* Services and applications configured to launch automatically
+
+If there is confirmation of an incident, the IRT formulates an overall response strategy and reports the incident to the Oversight Committee and other concerned parties, as appropriate.
+
+##### Phase 3: Containment, Eradication, and Recovery
```

Comment:
I've found it useful in the past to explicitly call out that folks are empowered to take, at time drastic measures to close holes. For example, in the rare situation that there is evidence that data is being actively exfiltrated, shutting down the entire service might be appropriate.

Reply:
As an enhancement to our IR processes, we'll develop playbooks that address specific types of security incidents. I think this information is a great candidate for inclusion in those playbooks. I'd like to keep the IRP itself quite high-level so that information within doesn't overlap with the playbooks.